vector/src/main/java/im/vector/app/features/roommemberprofile/RoomMemberProfileController.kt

```diff
@@ -121,8 +121,9 @@ class RoomMemberProfileController @Inject constructor(
                             editable = true,
                             icon = icon,
                             tintIcon = false,
-                            divider = false,
-                            action = { callback?.onShowDeviceList() }
+                            divider = false
+                            // Tchap: disable onClick for "Learn more" part
+//                            action = { callback?.onShowDeviceList() }
```

Comment:
After thinking, I realized disable the click is not enough, we could not keep this item "Verify/Learn more" visible.
I updated the request in #273 : " hide the item "Vérifier" or "en savoir plus" dans la section "Sécurité""